symbols/pdb/Test/Mono.Cecil.Pdb.Tests.csproj

```diff
@@ -1,30 +1,22 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\..\Tests.props" />
   <PropertyGroup>
-    <ProjectGuid>{29300103-CB76-4A1D-B6FD-FFD91C1EC8AA}</ProjectGuid>
+    <TargetFrameworks>netcoreapp2.1;net35;net40</TargetFrameworks>
     <RootNamespace>Mono.Cecil.Pdb.Tests</RootNamespace>
     <AssemblyName>Mono.Cecil.Pdb.Tests</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Mono.Cecil.csproj">
-      <Project>{D68133BD-1E63-496E-9EDE-4FBDBF77B486}</Project>
-      <Name>Mono.Cecil</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\..\Test\Mono.Cecil.Tests.csproj">
-      <Project>{A47B1F49-A81A-43E8-BE6B-DD28AF2C4055}</Project>
-      <Name>Mono.Cecil.Tests</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Mono.Cecil.Pdb.csproj">
-      <Project>{63E6915C-7EA4-4D76-AB28-0D7191EEA626}</Project>
-      <Name>Mono.Cecil.Pdb</Name>
-    </ProjectReference>
+    <ProjectReference Include="..\..\..\Mono.Cecil.csproj" />
+    <ProjectReference Include="..\..\..\Test\Mono.Cecil.Tests.csproj" />
+    <ProjectReference Include="..\Mono.Cecil.Pdb.csproj" />
+    
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Mono.Cecil.Tests\*.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="Resources\**\*" />
+    <None Include="Resources\**\*" >
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Why do we need to copy the Resources to the output directory?

Reply:
I don't remember but I think if I did it it's because it was needed.